firebase-firestore/src/test/resources/json/limbo_spec_test.json

```diff
@@ -3638,15 +3627,21 @@
                   "orderBys": []
                 }
               ],
-              "resumeToken": ""
+              "resumeToken": "resume-token-1000000"
             }
           }
         },
-        "clientIndex": 0
+        "clientIndex": 1
```

Comment:
It looks like the generator isn't generating a stable order among client indexes?

Reply:
I'll look into this and reply back.  There must have been a change between now and November 2019 (4 months ago) that affected this as the spec tests have not been regenerated since then.

Reply:
Update: The culprit is https://github.com/firebase/firebase-js-sdk/pull/2693, discovered by performing a bisection.  I'm continuing to investigate.

Reply:
It looks like the change in the spec test is expected; however, in the culprit PR the spec tests for Android (and probably iOS) were not updated accordingly.  I will create a separate PR to bring the spec tests up-to-date from the PR so that this PR can encompass only a single update to the spec tests.

Reply:
I've created a PR, https://github.com/firebase/firebase-js-sdk/pull/2843, to produce stable JSON from `generate_spec_json.js`.  I will hold off merging this PR until that PR is resolved so that a commit with a noisy diff in the spec tests can be limited to one commit.

Reply:
Update: commit 19ab883 was added to this PR with the stable JSON.